packages/host/app/components/host-mode/breadcrumb-item.gts

```diff
@@ -0,0 +1,160 @@
+import { on } from '@ember/modifier';
+import { action } from '@ember/object';
+
+import Component from '@glimmer/component';
+import { cached } from '@glimmer/tracking';
+
+import { cardTypeIcon } from '@cardstack/runtime-common';
+
+import { getCard } from '@cardstack/host/resources/card-resource';
+
+import type { ComponentLike } from '@glint/template';
+
+interface Signature {
+  Element: HTMLButtonElement;
+  Args: {
+    cardId: string;
+    disabled?: boolean;
+    isActionable?: boolean;
```

Comment:
Does “actionable” mean clickable? It seems unclear to me, I wouldn’t expect a breadcrumb to be actionable